docs/api/throttling.md

```diff
@@ -0,0 +1,33 @@
+# NuGet API rate limiting
+
+I order to prevent the extreme overload on NuGet Server resources the NuGet.org service API needs some configurable and enforceable upper limits on acceptable API usage. 
```

Comment:
~~I order to prevent the extreme overload on NuGet Server resources the NuGet.org service API needs some configurable and enforceable upper limits on acceptable API usage.~~ 

That's probably overly verbose.  I'd go with:

The NuGet.org API enforces rate limiting to prevent abuse. Requests that exceed the rate limit return the following error:

Reply:
Updated.